src/components/menuBar/menuBarButtons.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { SetStateAction, Dispatch } from "react";
+import styled from "@emotion/styled";
+import { Tag } from "../displayDocuments/documents";
+
+const Button = (props: { title: string; selected: boolean }): JSX.Element => {
+  const { title, selected } = props;
+  const ButtonContainer = styled.button`
+    line-height: 17px;
+    letter-spacing: -0.02em;
+    padding: 6px 12px;
+    border-radius: 0.5rem;
+  `;
+
+  const className = selected
+    ? "w-auto mx-1 font-semibold text-primary bg-primary-light focus:outline-none"
+    : "w-auto mx-1 font-normal text-gray-700 text-sm transition-colors duration-300 ease-out hover:bg-primary-light hover:text-primary";
+
+  return <ButtonContainer className={className}>{title}</ButtonContainer>;
+};
+
+interface MenuBarButtonsProps {
+  selectedButton: Tag | null;
+  setSelectedButton: Dispatch<SetStateAction<Tag | null>>;
+}
+
+export const MenuBarButtons = (props: MenuBarButtonsProps): JSX.Element => {
```

Comment:
Same, can destructure here.

```suggestion
export const MenuBarButtons = ({setSelectedButton, selectedButton }: MenuBarButtonsProps): JSX.Element => {
```